src/eos-companion-app-integration-helper.c

```diff
@@ -470,6 +471,7 @@ lookup_eknservices_version (const gchar        *runtime_version,
     const gchar *eknservices_name;
     const gchar *search_provider_name;
   } runtime_to_eknservices_versions[] = {
+    { "1", "EknServices", "SearchProviderV1" },
```

Comment:
Is this really desired? Quote/Word of the day both are against SDK1, also Vroom and the Service is not able to show the content using EKNServices1, if I set the runtime in the vroom manifest to 2 it works.

Word and Quote of the day we should probably blacklist.

Reply:
I'm on the fence about blacklisting those two. Technically we have a working content app for both of them, but they aren't really meant to be user facing.

Since both of them are actually set as `NoDisplay=True` on the desktop side, I wonder if we should use that as the criteria to remove them from `/list_applications`. Thoughts @cosimoc ?

Reply:
> Vroom and the Service is not able to show the content using EKNServices1, if I set the runtime in the vroom manifest to 2 it works.

That is very curious. How did it work before if Vroom was an SDK1 app? It couldn't have been supported with EknServices2 ....